site/src/modules/dashboard/Navbar/Navbar.tsx

```diff
@@ -25,6 +26,9 @@ export const Navbar: FC = () => {
 		featureVisibility.audit_log && permissions.viewAnyAuditLog;
 	const canViewConnectionLog =
 		featureVisibility.connection_log && permissions.viewAnyConnectionLog;
+	// TODO: We should have a separate permission for this
+	const canViewAIGovernance =
```

Comment:
currently the permission check is `system.read`. we should be asserting that here.

it's definitely not the right permission long term, and we should have a separate permission, but there is still a "correct" permission check for today.

I also see rumblings of an `aibridge_interception` resource type. https://github.com/coder/coder/blob/9d09f5ada65abff0fc3628a58a0933cb1daf3b02/coderd/rbac/object_gen.go#L18-L25